build.sbt

```diff
@@ -34,50 +34,24 @@ lazy val root = (project in file("."))
   .settings(noPublishSettings)
   .settings(name := "Trace4Cats")
   .aggregate(
-    base,
-    `base-io`,
-    `base-laws`,
-    core,
+    `fp-utils`,
+    `fp-utils-io`,
+    `fp-utils-laws`,
     `dynamic-sampling`,
     `dynamic-sampling-config`,
-    `exporter-common`,
-    `exporter-stream`,
     filtering,
     fs2,
-    inject,
     `inject-io`,
```

Comment:
This one should probably be renamed

Reply:
Any suggestions, maybe something that aligns with a new package name that we'll select as part of #744?

Reply:
What do you think about just having a single `io` module that encompasses inject-io and fp-utils-io?

Reply:
I don't see why not, as we discussed elsewhere the only reason for having the separate `fp-utils` was because we want to keep the concerns of general purpose contextual effect manipulation separate from the rest of t4c, but that doesn't apply here